examples/glade-cmip6.json

```diff
@@ -0,0 +1,136 @@
+{
+  "stac_version": "0.9.0",
+  "stac_extensions": [
+    "collection-assets",
+    "https://github.com/NCAR/esm-collection-spec/tree/master/schema.json"
+  ],
+  "id": "glade-cmip6",
+  "title": "CMIP6 Archive on NCAR's GLADE",
+  "description": "This is an ESM collection for CMIP6 data accessible on the NCAR's GLADE filesystem",
+  "extent": {
+    "spatial": {
+      "bbox": [[-180, -90, 180, 90]]
+    },
+    "temporal": {
+      "interval": [[null, null]]
+    }
+  },
+  "providers": [
+    {
+      "name": " World Climate Research Programme",
+      "roles": ["producer", "licensor"],
+      "url": "https://www.wcrp-climate.org/wgcm-cmip/wgcm-cmip6"
+    },
+    {
+      "name": "National Center for Atmospheric Research (NCAR)",
+      "roles": ["host"],
+      "url": "https://www2.cisl.ucar.edu/resources/cmip-analysis-platform"
+    }
+  ],
+  "license": "proprietary",
+  "links": [
+    {
+      "href": "https://pcmdi.llnl.gov/CMIP6/TermsOfUse/TermsOfUse6-1.html",
+      "type": "text/html",
+      "rel": "license",
+      "title": "CMIP6: Terms of Use"
+    }
+  ],
+  "assets": {
+    "thumbnail": {
+      "href": "cmip6-thumbnail.png",
```

Comment:
Is this a relative link? I have used https://www.carbonbrief.org/wp-content/uploads/2019/11/cmip6_mips.jpg in the past.